group4/81966_Georgi_Gerginov/IT_Department/data_structures/ordered_set/ordered_set.cpp

```diff
@@ -0,0 +1,284 @@
+#ifndef ORDERED_SET_CPP
+#define ORDERED_SET_CPP
+#include <iostream>
+
+template <class T>
+class OrderedSet {
+
+	struct Node {
+		T data;
+		Node* next;
+		Node* prev;
+
+		Node(const T&, Node* = nullptr, Node* = nullptr);
+	};
+
+	size_t m_length;
+	Node* m_start;
+	Node* m_end;
+
+	void del();
+	void copy(const OrderedSet<T>&);
+	void swap(OrderedSet<T>&);
+
+	bool is_unique(const T&) const;
+
+	void push_front(const T&);
+	void push_back(const T&);
+
+public:
+	class Iterator {
+
+        Node* curr;
+        Node* start;
+		Node* end;
```

Comment:
looks good on my end

Reply:
It's different cause one is done with spaces, the other with tabs. Usually IDE's can be configured to put spaces (4 for example) when pressing tab to avoid this.

Reply:
I've configured mine to always place tabs so it shouldn't be that. I also checked it out the other day and it seemed okay but nonetheless I might re-do it later.